### PR TITLE
Guava 28

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Comment by suztomo** [comment]
Kokoro macos was stuck. Manually running the build.

File: boms/cloud-oss-bom/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>1.2.1-SNAPSHOT</version>
```
Comment (suztomo) [inline boms/cloud-oss-bom/pom.xml]: Guava major version changed. It should be `2.0.0`

Comment (elharo) [reply boms/cloud-oss-bom/pom.xml]: good catch. done.

**Comment by elharo** [comment]
FYI, we're seeing flaky timeouts in tests. We should keep an eye on this:

```
[ERROR] Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 11.542 s <<< FAILURE! - in com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest
[ERROR] testFindUpdates_beam(com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest)  Time elapsed: 10.867 s  <<< ERROR!
org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at org.apache.beam:beam-sdks-java-io-google-cloud-platform:jar:2.5.0 -> com.google.api:gax-grpc:jar:0.20.0 -> io.grpc:grpc-netty:jar:1.2.0
	at com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest.testFindUpdates_beam(DependencyGraphIntegrationTest.java:85)
Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for io.grpc:grpc-netty:jar:1.2.0
	at com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest.testFindUpdates_beam(DependencyGraphIntegrationTest.java:85)
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not transfer artifact io.grpc:grpc-netty:pom:1.2.0 from/to central (http://repo1.maven.org/maven2/): Operation timed out (Connection timed out)
	at com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest.testFindUpdates_beam(DependencyGraphIntegrationTest.java:85)
Caused by: org.eclipse.aether.transfer.ArtifactTransferException: Could not transfer artifact io.grpc:grpc-netty:pom:1.2.0 from/to central (http://repo1.maven.org/maven2/): Operation timed out (Connection timed out)
	at com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest.testFindUpdates_beam(DependencyGraphIntegrationTest.java:85)
Caused by: java.net.ConnectException: Operation timed out (Connection timed out)
	at com.google.cloud.tools.opensource.dependencies.DependencyGraphIntegrationTest.testFindUpdates_beam(DependencyGraphIntegrationTest.java:85)
```

**Pull request merged**